Feature: User removal from org (#65)
rapyuta-robotics/rapyuta-io-docs: hugo/content/3_how-tos/31_account-management/309_user-management.md

   - Projects
---

## Adding or Deleting Users from an Organization

{{%notice note%}}
 Only the  **Admin** of an organization can invite users to their organization.
{{%/notice %}}

To add or remove users to the rapyuta.io platform, do the following:

1. On the left navigation bar, click **Account > Organization**. The panel displays a list of users.

2. To add users:

   1. On **Users** tab, enter a valid email address of the user you want to invite to rapyuta.io.

   2. Click **ADD USER**.

   The newly invited user is added to the existing list of **USERS**. The **Account State** field displays the state of a user. A user can be in one of the following states:
   * **Invited**: The initial state of the user. The invited user will receive an invitation email to join rapyuta.io.
   * **Activated**: Once the invited user has registered and signed into rapyuta.io, their state automatically changes to **Activated**. Only an org admin can activate a deactivated user by selecting the activate icon under **Actions**. 
   * **Deactivated**: A user/organization admin can deactivate a user by selecting the deactivate icon under **Actions**.

>    * **Suspended**: A user can be in the suspended state due to non-payment.

Reviewer: Since we do not have anything to do with payments in rapyuta.io, we can say that
```
A user can be in a suspended state if their trial has ended
```
@krishnar90

   {{%notice info%}}

>    An org admin can view users in any state. Non-admins cannot view the users in invited state.

Reviewer: ```suggestion
   Users other than org admin cannot view the users in the invited state.
```

   {{%/notice%}}

3. To delete users:

>    1. Under **Actions**, click **Remove** against the user you want to remove. A confirmation page appears.

Reviewer: ```suggestion
   1. Under **Actions**, click **Remove** against the user you want to remove and you will see a confirmation pop-up.
```
